src/EditorFeatures/Test2/IntelliSense/IntellisenseQuickInfoBuilderTests.vb

```diff
@@ -392,6 +392,59 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
             AssertEqualAdornments(expected, container)
         End Sub
 
+        <WpfFact, Trait(Traits.Feature, Traits.Features.QuickInfo)>
+        <WorkItem(33001, "https://github.com/dotnet/roslyn/issues/33001")>
+        Public Async Sub QuickInfoShowsMethodReturns()
```

Comment:
No tests at the Quick Info API level, especially for the various cases around events/methods which you handled?